jwst/datamodels/schemas/multispec.schema.yaml

```diff
@@ -30,6 +30,8 @@ allOf:
               datatype: float64
             - name: BERROR
               datatype: float64
+            - name: NPIXELS
+              datatype: float64
```

Comment:
Does NPIXELS need to be a float? Are there cases where partial pixels are used?

Reply:
Yes, NPIXELS does need to be a float, because it is often the case that only a fraction of the first or last pixel in the extraction region will be included.